### PR TITLE
(DO NOT MERGE) - adds allele to variant page - no merge    AGR-1851

### PR DESCRIPTION
**Pull request by nathandunn**
adds variant to allelle page: 

https://agr-jira.atlassian.net/browse/AGR-1851

- [ ] needs gene genomeLocation data separately or on allele call
- [ ] move to the variant section (instead of own section)
- [x] add allele to variant page with filter with fake data

```
  // myl7
  // http://localhost:2992/gene/ZFIN:ZDB-GENE-991019-3#alleles-and-variants
  // az2
  // http://localhost:2992/allele/ZFIN:ZDB-ALT-181010-2
```

### CODE REVIEW

File: src/containers/allelePage/AlleleSequenceView.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <li>fmin : N/A fake {genomeLocation.start}</li>
+        <li>fmax: N/A  fake {genomeLocation.end}</li>
+        <li>strand: N/A fake {genomeLocation.end}</li>
+        <li>chromosome : N/A fake {genomeLocation.chromosome}</li>
+        <li>biotype: {allele.type}, though maybe mapped as a gene</li>
+        <li>symbol: {allele.symbol}</li>
+        <li>id: {allele.id}</li>
+        <li>species: {allele.species.name}</li>
+        <li>synonyms: {allele.synonyms}</li>
+      </ul>
```
Comment (paaatrick) [inline src/containers/allelePage/AlleleSequenceView.js]: What's all this? Debugging?

Comment (nathandunn) [reply src/containers/allelePage/AlleleSequenceView.js]: @paaatrick  sorry . . am still working on this (forgot to mark it) . . I noted some bugs in the viewer, though . .. 

This is kind of a demo for @sibyl229  for next week. 

I'll embed it in the variants section.

Comment (paaatrick) [reply src/containers/allelePage/AlleleSequenceView.js]: All good, thanks for clarifying

**Comment by nathandunn** [comment]
replaced by #582

**Pull request closed without merging**